apps/gpc-prover-client/app/util/generateProof.ts

```diff
@@ -64,7 +64,9 @@ const prove = async (
 
   // We need a URL for downloading GPC artifacts depending on the configuration in the browser.
```

Comment:
This could benefit from a bit of explanatory text on the two options here and why someone would use them.
Packing into your own server is best if you want to control availability.
Using unpkg (or jsdelivr) is easiest for testing, but could be subject to downtime and rate limits.